includes/class-wcpdf-settings.php

```diff
@@ -693,6 +711,132 @@ public function get_sequential_number_store_method() {
 
 		return $method;		
 	}
+	
+	public function schedule_yearly_reset_numbers() {
+		if ( ! $this->maybe_schedule_yearly_reset_numbers() ) {
+			return;
+		}
+		
+		// checks AS functions existence
+		if ( ! function_exists( 'as_schedule_single_action' ) || ! function_exists( 'as_get_scheduled_actions' ) ) {
```

Comment:
@alexmigf How about just using the WP cron function if the AS one doesn't exist? Though, is that really likely anyway? (The check is just an abundance of cautiousness?).

Reply:
> The check is just an abundance of cautiousness?

Yes, a bit, it's unlikely to fail, but we never know. I think if we commit with one solution, in this case AS, we should stay with it, because otherwise complicates the code and probably will never be used that way. For example, if AS is unavailable at the year change and a WP cron is created, then we should stick with it until the end of the year or AS will create a new one and we have then 2 simultaneous crons, or delete the WP cron and create an AS action to replace it. Anyway, I think it's unnecessary, even if it fails to create the AS action, the user could do it manually another time before the end of the year, when he realize and fix the AS issues.